### PR TITLE
ls: add Russian translation

### PR DESCRIPTION
**Pull request by NikonP**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages.ru/common/micro.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# micro
```
Comment (navarroaxel) [inline pages.ru/common/micro.md:1]: Please @NikonP remove this page from this PR.

File: pages/linux/aplay.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# aplay
```
Comment (navarroaxel) [inline pages/linux/aplay.md:1]: Please @NikonP remove this page from this PR.

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @NikonP, I left you some comment.

Remember that you can send all the translations in one PR, just keep 1 comment per page.
But in this PR you have pages from other PRs here.

**Review by navarroaxel — APPROVED** [review]
LTGM. Thanks @NikonP

**Review by schneiderl — APPROVED** [review]
Don't know the language but looks good to me.

**Pull request merged**